### DISCUSSION TITLE
Using GUI works but sending the same packet using CLI doesn't

### DISCUSSION THREAD

**Discussion by deanholtwork** [dannagle/PacketSender#258] [General]
Hello Dan,

I am trying to use PacketSender to send commands to a networked piece of AV equipment. I am using Windows 10 and if I use the GUI the commands works and the piece of AV equipment does what it is supposed to do. If I try the same command using CLI then it does not work. I have tried with and without the firewall enabled and using cli, the command still does not work. The network is IPv4. I have tried two different windows 10 machines and I still get the same issue. I haven't tried on a Linux machine yet.

I have attached screengrabs of the GUI and CLI for reference.

I am hoping this is something obvious that I have missed.

Thanks,
Dean

![gui - works](https://user-images.githubusercontent.com/1785745/133096394-89e6174e-aed7-4956-a1ac-560c3cb9aa12.png)
![cli - does not work](https://user-images.githubusercontent.com/1785745/133096399-447cbe66-44aa-4f0c-849c-f8a95578aeb1.png)

**Comment by dannagle** [comment]
The command line Packet Sender behaves slightly differently than the GUI. With some extra arguments, it can behave more like the GUI.

Here are a couple things to try. 

- Use `-taw 500` to tell Packet Sender to wait for a response even if you don't need it. Devices with embedded network stacks, such as Global Cache, can sometimes be slow to process, and the command line Packet Sender may exit too quickly.  The "t" means TCP. The "a" means mixed ascii notation (like the GUI). The "w" means wait for a reply.

Example:
```
packetsender -taw 500 IP_ADDRESS PORT_NUM "your ascii sendir command"
```


- Send the packet you previously saved in the GUI. You will probably still want to add the wait statement. The "n" means use a previously stored named packet.

Example:
```
packetsender -taw 500 -n "My named packet"
```

I hope that helps.

**Reply by deanholtwork** [reply]
Hi Dan,

Thanks for the prompt response.

The -w 500 option works perfectly using the CLI. I shall certainly make a note of this.

Thanks again.

Regards,
Dean
